src/App/Resources/en-US/Resources.resw

```diff
@@ -687,6 +695,10 @@ On retry</value>
     <value>total runs</value>
     <comment>The total number of times a "run" (of a test) has happened, ie "5 total runs".</comment>
   </data>
+  <data name="SettingsButton.[using:Windows.UI.Xaml.Automation]AutomationProperties.Name" xml:space="preserve">
+    <value>Settings</value>
+    <comment>Factory OS Settings</comment>

```

Comment:
Factory Orchestrator App settings and not for OS.

Reply:
Noted and fixed